lib/vault/api/auth_token.rb

```diff
@@ -24,7 +24,12 @@ class AuthToken < Request
     #
     # @return [Secret]
     def create(options = {})
-      json = client.post("/v1/auth/token/create", JSON.fast_generate(options))
+      url = if options.include?(:role)
+              "/v1/auth/token/create/" + options.delete(:role)
+            else
+              "/v1/auth/token/create"
+            end
+      json = client.post(url, JSON.fast_generate(options))
```

Comment:
👍  on the PR!

Would you need to drop the `:role` key from the `options` hash here too? As it's not a valid parameter. Does the `/v1/auth/token/create/` endpoint simply ignore it?